src/platform/plugins/shared/usage_collection/server/usage_counters/usage_counters_service.test.ts

```diff
@@ -190,14 +191,15 @@ describe('UsageCountersService', () => {
   });
 
   it('retries errors by `retryCount` times before failing to store', async () => {
+    const retryConst = 1;
     const usageCountersService = new UsageCountersService({
       logger,
-      retryCount: 1,
```

Comment:
Can we retry more than once to confirm the backoff behavior?

Reply:
Done: https://github.com/elastic/kibana/pull/206363/commits/bd691beb47e9b35fb679f379d6c67cf25eb61076